distributed/comm/core.py

```diff
@@ -238,7 +238,7 @@ def resolve_address(addr):
 
 
 @gen.coroutine
-def connect(addr, timeout=3, deserialize=True):
+def connect(addr, timeout=3, deserialize=True, ssl_options=None):
```

Comment:
We do have to add a transport-kind specific option of passing additional parameters around.  
Not sure where the best place for that is other than a **kwargs.

New to this codebase.